src/vivarium/framework/lookup/interpolation.py

```diff
@@ -8,13 +7,14 @@
 simulations.
 
 """
+from __future__ import annotations
 
-from typing import List, Tuple, Union
+from typing import Any, Union
 
 import numpy as np
 import pandas as pd
 
-_ParameterType = Union[List[List[str]], List[Tuple[str, str, str]]]
+_ParameterType = Union[list[list[str]], list[tuple[str, str, str]]]
```

Comment:
You definitely can use `|` even when defining types like here.

Reply:
I'm not sure we need this type. This is only used for `continuous_parameters` and that seems like it could be type-hinted as `list[Sequence[str]]` (note: not `list[Iterable[str]]`, since the internal sequence must be ordered). The input is validated to ensure it matches the appropriate structure (i.e. has exactly 3 elements). 

If you don't remove it, you should definitely make it not be private, since it will appear in the api docs.